forms-flow-web/src/components/Form/List.scss

```diff
@@ -118,6 +122,40 @@ flex-wrap: wrap;
     font-weight: normal
 }
 
+.btn-upload-fail{
+    width: 20%;
+}
+
+.form-check-input{
+    width: 15px;
+    height: 15px;
+}
+
+.form-upload{
+    padding-bottom: 7px;
+    color: #450ccb;
```

Comment:
did you check the hard-coded color changed by variable or not with @Josephalexantony-aot